js/package.json

```diff
@@ -25,6 +25,9 @@
     "typescript": "^5.1.3"
   },
   "dependencies": {
+    "@bufbuild/buf": "^1.32.2",
+    "@bufbuild/protobuf": "^1.10.0",
```

Comment:
Note: I made a comment above against `package-lock` about looking into switching a few of these to `devDeps`. Meant to make the comment here!

Reply:
Done!